conformance/utils/http/http.go

```diff
@@ -71,7 +72,17 @@ func MakeRequestAndExpectEventuallyConsistentResponse(t *testing.T, r roundtripp
 		expected.StatusCode = 200
 	}
 
-	t.Logf("Making %s request to http://%s%s", expected.Request.Method, gwAddr, expected.Request.Path)
+	// Parse gwAddr so we can tell which family it belongs to.
```

Comment:
Maybe we could handle this centrally in `WaitForGatewayAddress`

Reply:
We could, but "[2001:470:154:5::1]" isn't a valid IP address so I'd be hesitant to pass that around to code that might want to use it as an address. The square-brackets are part of the URL, not part of the IP address.

The good news is that this is the only place in the conformance tests that formats URLs.

Reply:
I think @michaelbeaumont's updated comment that #1095 should fix this makes sense, can you confirm @acnodal-tc?

Reply:
I'll give it a try.